src/tests/Interop/common/ComHelpers.h

```diff
@@ -80,9 +87,15 @@ class UnknownImpl
         }
         else
         {
+            //Internal::__QueryInterfaceImpl available only for _WIN32 due to __uuidof(T) availability
+#ifdef _WIN32            
             HRESULT hr = Internal::__QueryInterfaceImpl(riid, ppvObject, i1, remain...);
             if (hr != S_OK)
                 return hr;
+#else
+            *ppvObject = nullptr;
+            return E_NOINTERFACE;
```

Comment:
Since this is not doing anything I would instead return `E_NOTIMPL`.

Reply:
ok